Survey/Sources/Supports/Extensions/Foundation/String+Extensions.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  String+.swift
+//  String+Extensions.swift
```

Comment:
I suggest making it clearer by defining the extension's purpose
E.g:
```suggestion
//  String+Validation.swift
```
E.g from a previous project
<img width="260" alt="Screen Shot 2022-08-16 at 09 36 57" src="https://user-images.githubusercontent.com/24598204/184786375-421fe59c-fac4-43f0-8a8b-c3ab5b466751.png">

Reply:
Thank you for sharing your best practice 🙏 , I applied this rule in https://github.com/nkhanh44/ic-survey-ios/pull/18/commits/0093261ceda17685a647a3db1ade5c330fac9868